lib/schema/cbg.js

```diff
@@ -24,7 +24,7 @@ schema.registerIdFields('cbg', idFields);
 
 module.exports = schema.makeHandler('cbg', {
   schema: {
-    deviceTime: schema.validDeviceTime,
+    deviceTime: schema.ifExists(schema.validDeviceTime),
```

Comment:
is it worth a comment that this was relaxed for healthkit integration? Might not be but q popped into my mind so thougt i would put it out there